pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -31,10 +31,10 @@ type SNMPConfig struct {
 
 	//General
 	SnmpIPAddress string `yaml:"ip_address"`
-	SnmpPort      string `yaml:"port"`
+	SnmpPort      uint16 `yaml:"port"`
```

Comment:
Opinionated, these `snmp` prefix seem redundant. We know we are in snmp because the struct is called `SnmpConfig`

Reply:
Not a blocker though !